web/src/connectors/task-annotator-connector/task-annotator-context.tsx

```diff
@@ -210,12 +192,11 @@ export const TaskAnnotatorContextProvider: React.FC<ProviderProps> = ({
     const [selectionType, setSelectionType] = useState<
         AnnotationBoundType | AnnotationLinksBoundType | AnnotationImageToolType
     >('free-box');
-    const [selectedTool, setSelectedTool] = useState<AnnotationImageToolType>('pen');
+    const [selectedTool, setSelectedTool] = useState<AnnotationImageToolType>(ToolNames.pen);
     const [selectedAnnotation, setSelectedAnnotation] = useState<Annotation | undefined>();
-    const [isDataTabDisabled, setIsDataTabDisabled] = useState<boolean>(dataTabDefaultDisableState);
```

Comment:
how we disable Data tab now?

Reply:
We don't do it anymore as `isDataTabDisabled` is not using anywhere

Reply:
`isDataTabDisabled` wasn't using anywhere, so we don't need to disable it at all

Reply:
But we still need to disable Data tab if there is no selected annotation, right?

Reply:
As I said - we don't use `isDataTabDisabled`, we didn't use it before this refactoring and we don't use it now.

Please refer to place where we use `isDataTabDisabled`, I couldn't find any place with it

Reply:
Ok, I see. In that case could you please check how it disables now. If it works fine - ok, we can delete this, but if it doesn't work as expected, maybe we still need this code?

Reply:
It looks like we don't disable "Data" tab at all
<img width="643" alt="image" src="https://user-images.githubusercontent.com/40606828/233284950-7d0d1cf4-34f2-4005-85be-c366388d3a15.png">

Also I don't think that we need to keep not used code that may be useful at some point in the future, we can return it if we need to

Reply:
Because code which may be useful at some point in future === useless code which definitely won't be useful in future

Because of such approach we will have garbage in code-base
And about `If you remove this now and other developer will need to fix it` - it's not a problem to implement it again, but having garbage in code base is a problem